cmd/login.go

```diff
@@ -110,45 +122,19 @@ type loginCmdArgs struct {
 }
 
 func (lca loginCmdArgs) validate() error {
```

Comment:
TODO : This function def needs to change to func (lca *loginCmdArgs) validate() error {